crates/bellman/src/plonk/better_better_cs/proof/mod.rs

```diff
@@ -657,6 +657,7 @@ impl<E: Engine, P: PlonkConstraintSystemParams<E>, MG: MainGate<E>, S: Synthesis
             grand_products_protos_with_gamma.push(p);
         }
 
+        #[allow(clippy::redundant_locals)]
         let required_domain_size = required_domain_size;
```

Comment:
why?

Reply:
I prefer not to touch the code that I'm not confident with. `redundant_locals` may have false positives related to scopes and moves. It doesn't look to be the case here, but I decided to play it safe.

Reply:
Btw I like the name `better_better_cs`

Reply:
The next one will surely be the `best`.